docs/de/README.md

```diff
@@ -53,8 +53,9 @@ und soll Angaben zur Installation und ggf. zum Erwerb eines Lizenzschlüssels et
 
 ## Rechtschreibprüfung
 
-Die Metadaten werden automatisch auf korrekte Rechtschreibung überprüft. Womöglich musst du die `whitelist.txt` aktualisieren,
-wenn die Überprüfung fehlschlägt, du dir aber sicher bist, dass das Wort korrekt ist.
+Die Metadaten werden automatisch auf korrekte Rechtschreibung überprüft. Wenn die Überprüfung fehlschlägt, du dir aber sicher bist, 
+dass das Wort korrekt ist.musst du womöglich die Whitelists aktualisieren. Diese werden im Ordner `linter/whitelists` nach Sprache 
```

Comment:
Can you fix the spelling issue here `ist, musst` and is there any reason why this is on a new line?

Reply:
And one final rebase, then I can merge :) Thanks David!

Reply:
Fixed the typo. I used hard line breaks because of consistency. The whole readme does it (though I'm not sure what you use as line limit).

Reply:
I rebased it again. Are you able to merge it now?